EIPS/eip-3855.md

```diff
@@ -4,7 +4,7 @@ title: PUSH0 instruction
 description: Introduce a new instruction which pushes the constant value 0 onto the stack
```

Comment:
Unrelated to this PR: We need to get the eips.ethereum.org site updated to render this field.

Reply:
I lied, it *is* included, just below the title rather than in the table.

Reply:
Yeah, remember #3706 where that option was most liked? :)